### ISSUE TITLE
Autoconf template has errors (with autoconf 2.69)

### ISSUE THREAD

**Issue by sanmai-NL** [proycon/ucto#1]
I'm at dd2f374139b431771a3bf28b00f42e4870342a81 on origin/master.

```fish
sander@Yoga:/opt/installers/ucto/ucto/  autoconf
configure.ac:8: error: possibly undefined macro: AM_INIT_AUTOMAKE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:31: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:74: error: possibly undefined macro: AC_MSG_FAILURE
```
```fish
sander@Yoga:/opt/installers/ucto/ucto/  autoconf -V
autoconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.
```

**Comment by proycon** [comment]
I think this is caused by missing the Autoconf Macro Archive. Please install the autoconf-archive package if you're on Debian/Ubuntu, it may be named ac-archive on certain other Linux/BSD distributions.

**Comment by sanmai-NL** [comment]
Thanks. I'm on Arch Linux, so I issued:
```sh
sudo pacman -S autoconf-archive
```
and that (at version 1:2015.02.24-1, plus the `automake-1.15.1` and `libtool-2.4.6-1` packages) are now definitely installed.

However, it does not work as is:
```
sander@Yoga:/opt/installers/ucto/ucto/  git reset --hard origin/master
HEAD is now at dd2f374 updated, modernized
```
```
sander@Yoga:/opt/installers/ucto/ucto/  autoconf
```
```
sander@Yoga:/opt/installers/ucto/ucto/  
./configure --prefix=$INSTALLATIONS/ucto/ucto-git
./configure: line 2027: AM_INIT_AUTOMAKE: command not found
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
./configure: line 2557: AC_PROG_LIBTOOL: command not found
./configure: line 2558: LT_INIT: command not found
checking how to run the C++ preprocessor... g++ -E
checking for grep that handles long lines and -e... /usr/bin//grep
checking for egrep... /usr/bin//grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... no
checking for inline... inline
checking for size_t... yes
checking for int32_t... yes
./configure: line 3146: syntax error near unexpected token `4.2,'
./configure: line 3146: `  AX_ICU_CHECK( 4.2,'
```
Do you have a particular reason to use GNU autotools?

**Comment by proycon** [comment]
I'm on Arch myself too. It seems the error is simply caused by running autoconf directly rather than the provided "bootstrap.sh" script, which calls aclocal, automake and autoconf for you. Perhaps the instructions should be updated to clarify this.

**Comment by proycon** [comment]
By the way, the latest 'stable' version of ucto is also in the AUR, it will be a bit behind the git version though.

**Comment by sanmai-NL** [comment]
Thanks! 
Running bootstrap.sh worked okay, even without `bash` (`sh`: `dash`, simpler more standard shell). Too bad I needed `libfolia`, and then the as yet undocumented `ticcutils` library. 

But your Arch package is indeed far more convenient, I can confirm that installing that went fine for me (current Arch, same as you probably).  By the way, using Google I ended up here, as the availability of  Arch packages is still omitted on http://ilk.uvt.nl/ucto/ and other official pages I encountered.
